### ISSUE TITLE
New Site Arrangement

### ISSUE THREAD

**Issue by Mulan-94** [ratt-ru/ratt-website#7]
### This is the landing page. Doesn't look extra to me. Any ideas?
![image](https://user-images.githubusercontent.com/12611059/114531772-7d62b300-9c4c-11eb-9551-e169a2a46aff.png)
![image](https://user-images.githubusercontent.com/12611059/114531853-8e132900-9c4c-11eb-8748-3a76902e8fb3.png)

### I'm not too sure about this look  for the different sub categories. This is how the theme lists them
![image](https://user-images.githubusercontent.com/12611059/114531929-a3885300-9c4c-11eb-8ba2-e42d788aae56.png)

![image](https://user-images.githubusercontent.com/12611059/114531999-b26f0580-9c4c-11eb-9025-234cc16bcfd3.png)

### People's theses
![image](https://user-images.githubusercontent.com/12611059/114532089-c74b9900-9c4c-11eb-8d21-23594164cc43.png)

**Comment by o-smirnov** [comment]
Actually, this theme is growing on me, nice and clean. We could try some radio maps with funky colour scales to give it some extra zing?

Some niggles:

* The extra "RATT" under the logo is redundant (and grates on me because it's the same word in a different font)

* Is there a way to display only certain categories of content under "Recent posts" (let's call it Recent news...)? Things like job ads etc. ought to show up there, but not staff pages

* The "Categories" box is not terribly useful on the front page as it is. Can we display it only on certain sub-pages? Else not at all.

Which branch is it on?

**Comment by Mulan-94** [comment]
The branch is **slim-theme** https://github.com/ratt-ru/ratt-website/tree/slim-theme
> The extra "RATT" under the logo is redundant (and grates on me because it's the same word in a different font)

Ok will change it

>Is there a way to display only certain categories of content under "Recent posts" (let's call it Recent news...)? Things like job ads etc. ought to show up there, but not staff pages
> The "Categories" box is not terribly useful on the front page as it is. Can we display it only on certain sub-pages?

I think yes, I just need to find it

**Comment by o-smirnov** [comment]
```
oms@tshikovski:~/projects/ratt-website$ hugo server -D
Error: Unable to locate config file or config directory. Perhaps you need to create a new site.
       Run `hugo help new` for details.
```
Am I missing a symlink or something?

**Comment by o-smirnov** [comment]
Yeah, found it, ``config.toml`` needed to be symlinked to a theme.

**Comment by Mulan-94** [comment]
I have been running it with: `hugo --config slim_config.toml server -D --port 5000 --disableFastRender --ignoreCache --noHTTPCache `

My setup details are in the readme

**Comment by Mulan-94** [comment]
use **slim_config.toml**

**Comment by Mulan-94** [comment]
## Added a few royalty free and meerkat/rarg group images now
![image](https://user-images.githubusercontent.com/12611059/114566679-ce859d80-9c72-11eb-9cb8-8314dd5dde75.png)
t.com/12611059/114566491-9f6f2c00-9c72-11eb-8ebc-8e6857e6db95.png)

![image](https://user-images.githubusercontent.com/12611059/114566880-ff65d280-9c72-11eb-8f42-2b3e58c38fa8.png)

**Comment by o-smirnov** [comment]
I like that last one. Especially the "coronavirus" plot, very trendy. :)
